src/DateTimeRoutines/DateTimeRoutines.cs

```diff
@@ -285,10 +285,7 @@ public static bool TryParseDateTime(this string str, DateTimeFormat default_form
         /// <param name="default_format">format to be used preferably in ambivalent instances</param>
         /// <param name="parsed_time">parsed date-time output</param>
         /// <returns>true if time was found, else false</returns>
-        public static bool TryParseTime(this string str, DateTimeFormat default_format, out ParsedDateTime parsed_time)
-        {
-            return TryParseTime(str, default_format, out parsed_time, null);
-        }
+        public static bool TryParseTime(this string str, DateTimeFormat default_format, out ParsedDateTime parsed_time) => TryParseTime(str, default_format, out parsed_time, null);
```

Comment:
I agree with this PR, but what is the maximum line length? We should set one limit to avoid really long lines like this one.
```
public static bool TryParseTime(this string str, DateTimeFormat default_format, out ParsedDateTime parsed_time)
        => TryParseTime(str, default_format, out parsed_time, null);
```

Reply:
There currently isn't one. I'll find the Microsoft recommended length and add it to the next one